<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompatOptsDialog</class>
 <widget class="QDialog" name="CompatOptsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Detailed compatibility options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="actorGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Actor behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="crushedMonsters">
            <property name="text">
             <string>Crushed monsters can be resurrected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="friendlyMonsters">
            <property name="text">
             <string>Friendly monsters aren't blocked</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="limitPainElem">
            <property name="text">
             <string>Limit Pain Elementals' Lost Souls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="monsterMovement">
            <property name="text">
             <string>Monster movement is affected by effects</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="monstersCannotCross">
            <property name="text">
             <string>Monsters cannot cross dropoffs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="monstersGetStuck">
            <property name="text">
             <string>Monsters get stuck over dropoffs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="monstersSeeInvisible">
            <property name="text">
             <string>Monsters see invisible players</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="noMinotaurFloor">
            <property name="text">
             <string>No Minotaur floor flames in water</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="spawnItemDrops">
            <property name="text">
             <string>Spawn item drops on the floor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dehackedGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>DeHackEd behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="dehHealthSettings">
            <property name="text">
             <string>DEH health settings like Doom2.exe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="originalAMushroom">
            <property name="text">
             <string>Original A_Mushroom speed in DEH mods</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mapActionGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Map/action behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="allSpecialLines">
            <property name="text">
             <string>All special lines can block&lt;use&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="allowAnyBossdeath">
            <property name="text">
             <string>Allow any bossdeath for level special</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="disableBoomDoor">
            <property name="text">
             <string>Disable Boom door light effect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="findNeighboringLight">
            <property name="text">
             <string>Find neighboring light like Doom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="findShortestTextures">
            <property name="text">
             <string>Find shortest textures like Doom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useBuggierStair">
            <property name="text">
             <string>Use buggier stair building</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useDoomsFloor">
            <property name="text">
             <string>Use Doom's floor motion behavior</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useDoomsPointOnLine">
            <property name="text">
             <string>Use Doom's point-on-line algorithm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="levelExit">
            <property name="text">
             <string>Level exit can be triggered more than once</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="physicsGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Physics behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="actorsAreInfinite">
            <property name="text">
             <string>Actors are infinitely tall</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="boomScrollers">
            <property name="text">
             <string>Boom scrollers are additive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cannotTravelStraight">
            <property name="text">
             <string>Cannot travel straight NSEW</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableWallRunning">
            <property name="text">
             <string>Enable wall running</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ravenScrollers">
            <property name="text">
             <string>Raven scrollers use original speed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="selfRefSectors">
            <property name="text">
             <string>Self-referencing sectors don't block shots</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useDoomHitscan">
            <property name="text">
             <string>Use Doom code for hitscan checks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useDoomHeights">
            <property name="text">
             <string>Use Doom heights for missile clipping</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="renderingGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Rendering behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="drawPolyobjects">
            <property name="text">
             <string>Draw polyobjects like Hexen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignoreYoffsets">
            <property name="text">
             <string>Ignore Y offsets on masked midtextures</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="invertSpriteSorting">
            <property name="text">
             <string>Invert sprite sorting</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="soundGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Sound behaviour</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="crippleSound">
            <property name="text">
             <string>Cripple sound for silent BFG trick</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dontLetOthersHear">
            <property name="text">
             <string>Don't let others hear your pickups</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="instantMovingFloors">
            <property name="text">
             <string>Instant moving floors are not silent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sectorSounds">
            <property name="text">
             <string>Sector sounds use center as source </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="soundsStop">
            <property name="text">
             <string>Sounds stop when actor vanishes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useOriginalSound">
            <property name="text">
             <string>Use original sound target handling</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="compatflags1_label">
       <property name="text">
        <string>compatflags1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compatflags1_line"/>
     </item>
     <item>
      <widget class="QLabel" name="compatflags2_label">
       <property name="text">
        <string>compatflags2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compatflags2_line"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CompatOptsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CompatOptsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
